src/modules/hyprland/workspaces.cpp

```diff
@@ -56,55 +68,73 @@ auto Workspaces::update() -> void {
     create_workspace(workspace_to_create);
   }
 
-  workspaces_to_create_.clear();
-
-  for (std::unique_ptr<Workspace> &workspace : workspaces_) {
-    workspace->set_active(workspace->id() == active_workspace_id);
-
+  for (auto &workspace : workspaces_) {
+    workspace->set_active(workspace->name() == active_workspace_name);
```

Comment:
It's better to leave it with integers comparision.

Reply:
same (IPC gives name not id)